### PR TITLE
update using-native-node-modules doc

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: docs/tutorial/using-native-node-modules.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ where to download headers and which version to use:
 
 ```bash
 $ cd /path-to-module/
-$ HOME=~/.electron-gyp node-gyp rebuild --target=0.16.0 --arch=ia32 --dist-url=https://atom.io/download/atom-shell
+$ HOME=~/.electron-gyp node-gyp rebuild --target=0.25.0 --arch=ia64 --dist-url=https://atom.io/download/electron
```
Comment (zcbenz) [inline docs/tutorial/using-native-node-modules.md]: The dist-url is unchanged to keep compatibility with existing tools.

File: docs/tutorial/using-native-node-modules.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ You can also use `npm` to install modules, the steps are exactly the same with
 Node modules, except that you need to setup some environment variables:
 
 ```bash
-export npm_config_disturl=https://atom.io/download/atom-shell
-export npm_config_target=0.23.0
+export npm_config_disturl=https://atom.io/download/electron
```
Comment (zcbenz) [inline docs/tutorial/using-native-node-modules.md]: Same here.

**Comment by MaxGraey** [comment]
Ок, I revert to atom-shell url. What about the rest?

**Comment by zcbenz** [comment]
The rest looks good to me, thanks!

**Pull request merged**